libs/dataviews-client/src/resolve-dataviews-generators.ts

```diff
@@ -402,6 +402,16 @@ export function getGeneratorConfig(
           (dataview.config?.type === GeneratorType.UserContext ||
             dataview.config?.type === GeneratorType.UserPoints)
         ) {
+          const circleRadiusProperty = dataset.configuration?.configurationUI?.pointSize
+
+          generator.circleRadiusProperty =
```

Comment:
As we talked I'm going to move the `getDatasetConfigurationProperty` to a new lib called `datasets-client` to be able to use it out of the map